packages/catalog-model/src/types.ts

```diff
@@ -34,3 +34,31 @@ export type EntityPolicy = {
 };
 
 export type JSONSchema = JSONSchema7 & { [key in string]?: JsonValue };
+
+/**
+ * A complete entity name, with the full kind-namespace-name triplet.
+ */
+export type EntityName = {
```

Comment:
What do you think about calling this `EntityHandle` instead? Or flip it around, make this the `EntityRef`, and the incomplete and/or generic form be `EntityHandle` or `EntityRefInput` or smth instead?

Current form is ofc quite easy to confuse with the name itself :grin: